ccmlib/scylla_node.py

```diff
@@ -1041,11 +1044,19 @@ def run_patchelf(patchelf_cmd):
                                    os.path.join(self.get_bin_dir(), 'scylla-jmx-1.0.jar'), replace=replace)
             self.hard_link_or_copy(os.path.join(self.get_jmx_dir(), 'scylla-jmx'),
                                    os.path.join(self.get_bin_dir(), 'scylla-jmx'), replace=replace)
+            select_java = Path(self.get_jmx_dir()) / 'select-java'
+            if select_java.exists():
+                self.hard_link_or_copy(select_java,
+                                       Path(self.get_bin_dir()) / 'select-java', replace=replace)
```

Comment:
I don't understand what this code does. You are making a copy of select-java itself? Why? Also, does the _start_jmx() above actually use this copy, or the original one?

Reply:
It uses the copy,

It's common across ccm that we copy the artifacts we use into the cluster code

We could just symlink it, in this case should be the same